pkg/kubelet/apis/config/types.go

```diff
@@ -88,6 +88,8 @@ type KubeletConfiguration struct {
 	// staticPodPath is the path to the directory containing local (static) pods to
 	// run, or the path to a single static pod file.
 	StaticPodPath string
+	// podLogsDir is a custom root directory path kubelet will use to place pod's log files.
```

Comment:
from functionality perspective, many things may work unexpectedly if user will supply temp folder as a log dir path. And since it's log directory, some may be tempted to use the temp folder for this.

I'd suggest we at a minimum mention it in description that temp folder is not a good idea to use as a log path.

Reply:
Updated